### ISSUE TITLE
Unable to rotate assets + cursor frequently disappearing

### ISSUE THREAD

**Issue by CalNie** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#192]
**Describe the bug**
Not quite sure how it happened, but I am unable to rotate any assets and I cannot remove a current asset without pressing ESC instead of right-clicking. Also, trying to use the click-and-drag function causes my cursor to vanish until I press ESC.

**To reproduce**
To be perfectly honest, I don't know what I did to cause this. I was playing around with the Cobblestones assets when it started happening. Persisted even after switching to Caverns and Tavern assets.

**Comment by W1LDxC4RD** [comment]
I can't rotate assets at all. Haven't ever been able to, unless it's that I'm missing something in the help menu. All I'm able to do is rotate the board itself.

**Comment by CalNie** [comment]
> I can't rotate assets at all. Haven't ever been able to, unless it's that I'm missing something in the help menu. All I'm able to do is rotate the board itself.

My bug has been fixed. In order to rotate assets, you have to click the middle mouse button. There is no smooth rotation as of yet that I know of, but I'm sure it won't be far behind.

**Comment by W1LDxC4RD** [comment]
> > I can't rotate assets at all. Haven't ever been able to, unless it's that I'm missing something in the help menu. All I'm able to do is rotate the board itself.
> 
> My bug has been fixed. In order to rotate assets, you have to click the middle mouse button. There is no smooth rotation as of yet that I know of, but I'm sure it won't be far behind.

You're right. Not sure if it's my mouse or the program, but it was tough to rotate them. Had to click pretty hard and sometimes multiple times to get them to rotate, but I finally got it to work. Thank you.
